it/topics/README.md

```diff
@@ -1,5 +1,22 @@
-# Il soggetto
+# 💡 Argomenti
```

Comment:
Can you make sure this title is the same here and in the it/README table?

Reply:
@MichaelCurrin, Sorry, I've just realized it. So I've fixed in #32